modules/start_on_team_color/functions/SOTC/fn_SOTC_Init.sqf

```diff
@@ -0,0 +1,17 @@
+#include "script_component.hpp"
+
+if (!isDedicated) then {
+  #include "..\..\settings.sqf"
+
+  "" spawn {
```

Comment:
This can be replaced by execNextFrame or a waitUntilAndExecute with !isnull ace_player

Reply:
Done.